boot/kernel_os.go

```diff
@@ -38,14 +37,8 @@ func RemoveKernelAssets(s snap.PlaceInfo) error {
 		return fmt.Errorf("no not remove kernel assets: %s", err)
 	}
 
-	// remove the kernel blob
-	blobName := filepath.Base(s.MountFile())
-	dstDir := filepath.Join(loader.Dir(), blobName)
-	if err := os.RemoveAll(dstDir); err != nil {
-		return err
-	}
-
-	return nil
+	// ask bootloader to remove the kernel assets if needed
+	return loader.RemoveKernelAssets(s)
```

Comment:
mmh, other places in the file  shadow bootloader and use a that name

bootloader, err := bootloader.Find()

not a fan of just "loader" for this

Reply:
seems a more general minor problem that I didn't spot so far.  anyway either "bl" or "bootloader" are better than just "loader"

Reply:
@pedronis changed to bootloader now to be aligned with use in other functions